app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -103,12 +104,22 @@
                 android:text="@string/sign_up"
                 android:textColor="#FFFFFF" />
 
+
             <TextView
-                android:id="@+id/text_view_sign_in"
+                android:id="@+id/text_view_account_already"
                 android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
                 android:layout_gravity="center"
+                android:text="Already have an account?" />
+
+            <TextView
+                android:id="@+id/text_view_sign_in"
+                android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:text="@string/have_account_text" />
+                android:layout_gravity="center"
+                android:text="Sign in here"
```

Comment:
This should be in the String.xml file.

Also, the test is failing because of this string. 

![image](https://user-images.githubusercontent.com/23643192/113585902-d69e6700-95fa-11eb-8c40-227de67339e6.png)

Either fix the test, or change this string back to what it was.

Reply:
ok, fixed